src/js/merkle-drop/api/web3.js

```diff
@@ -38,8 +34,46 @@ export async function claimTokens(
   }
 }
 
+export async function getClaimedTokenAmountByReceipt(receipt) {
+  const merkleDropContract = getMerkleDropContract()
+  const eventFilter = {
+    fromBlock: receipt.blockNumber,
+    transactionHash: receipt.transactionHash,
```

Comment:
I did not find in the documentation that you can use that? Where did you find that?

Reply:
To be honest I could not remember anymore, but could not find it myself again. Apparently that gets accepted. It works in the tests, but I don't trust the `web3js` implementation for that. So I removed it and do this part of the filter locally. Having the withdraw event for a single block it should not be much work. By that I took the opportunity and tried to produce more clean code.

Reply:
I tested it again with a deployment on _Kovan_.